src/app/services/raiden.service.ts

```diff
@@ -349,25 +349,14 @@ export class RaidenService {
             );
     }
 
-    public registerToken(tokenAddress: string): Observable<UserToken> {
+    public registerToken(tokenAddress: string): Observable<any> {
```

Comment:
```suggestion
    public registerToken(tokenAddress: string): Observable<void> {
```
Maybe you should properly convey intention since now we don't return any value from this observable.

Reply:
I was doing it this way because the other functions of the raiden service, which don't return values, behave the same. But your proposal makes absolutely sense. So I should change the other functions too, right?

Reply:
Or actually at the moment it returns the new token network contract address. So do we not want it to be returned, because we do not use anywhere?

Reply:
Yes for consistency it would be better to properly update the service API to reflect intent.